crates/sqlexec/src/planner/logical_plan/create_schema.rs

```diff
@@ -24,7 +24,16 @@ impl UserDefinedLogicalNodeCore for CreateSchema {
     }
 
     fn fmt_for_explain(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        write!(f, "CreateSchema")
+        write!(
+            f,
+            "CREATE SCHEMA{if_not_exists} {schema_reference}",
+            if_not_exists = if self.if_not_exists {
+                " IF NOT EXISTS"
+            } else {
+                ""
+            },
+            schema_reference = self.schema_reference
+        )
```

Comment:
originally added these to help debugging. Makes sense to keep them as they are better fmt strings anyways.

Reply:
1. Explain formatting shouldn't be the query string equivalent. If we want more information in the output, it should be in the form of `CreateSchema [refere='my_schema', if_not_exists=true]`. See other explain outputs for examples.
2. How were you using this for debugging. A `explain create schema my_schema` doesn't work.

Reply:
that makes sense. I'll revert these for now. 

> How were you using this for debugging. A explain create schema my_schema doesn't work.
I was just running the create table.slt, and there were several `create table` and `create schema` operations, I couldnt really tell which if any were failing, so I added these to try to narrow down which ones were failing.